### ISSUE TITLE
Some images are invisible in ODF->pdf converter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#100]
```
When a test file ODTBig.odt is converted to pdf, there are no images in the pdf 
file.
In the ODTBig.odt file images are in svm (Star View Metafile) format. iText 
converter does not recognize it.
open office sometimes wraps a bitmap in this format, this is an feature from 
"ancient" times.
I don't know if this problem is solvable at all. StarViewMetafile format is 
undocumented and there is no java library to handle it.

```

Original issue reported on code.google.com by `abe...@gmail.com` on 5 Apr 2012 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
Hi Leszek,

I know this problem. I don't know how we can solve it (but I have not done a 
big investigation).

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 5 Apr 2012 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
I have an idea - don't analyze the whole svm because it is a vector format
but search for a wrapped bitmap instead (by magic number). It must be somewhere 
in the file.
Leszek
```

Original comment by `abe...@gmail.com` on 5 Apr 2012 at 9:41

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.9.8
```

Original comment by `abe...@gmail.com` on 18 May 2012 at 11:21
- Changed state: **Fixed**
